src/gui/static/src/app/components/pages/exchange/exchange-status/exchange-status.component.ts

```diff
@@ -1,22 +1,32 @@
 import { Component, Input, OnDestroy, Output, EventEmitter } from '@angular/core';
-import { ExchangeOrder, StoredExchangeOrder } from '../../../../app.datatypes';
-import { ExchangeService } from '../../../../services/exchange.service';
 import { MatDialog } from '@angular/material/dialog';
 import { SubscriptionLike, of } from 'rxjs';
+import { delay, mergeMap } from 'rxjs/operators';
+
+import { ExchangeService, ExchangeOrder, StoredExchangeOrder } from '../../../../services/exchange.service';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/skycoin/skycoin/pullRequest?prid=7163600)
```suggestion
import { ExchangeService, ExchangeOrder, StoredExchangeOrder } from "../../../../services/exchange.service";
```